### PR TITLE
feat(design): add a `POST /participants` call and use more mojaloopy-style routing

### PR DESCRIPTION
**Pull request by lewisdaly**
- updates `POST /consents`, `POST /consents/{id}/generateChallenge` and `PUT /consents/{id}` calls to use Mojaloop style routing
- for `docs/linking/5b-credential-registration.puml`, switch to locally rendered image: I think its too big for the proxy renderer

Since the plant uml renderer won't update until merge, here's the updated diagrams:

PISP Linking  Grant consent

PISP Linking  Credential registration (challenge)

PISP Linking  Credential registration (verification)

### CODE REVIEW

File: docs/linking/README.md [hunk 1]
```diff
@@ -216,7 +228,12 @@ the signature is correct. It then updates the status of the credential to
 `"VERIFIED"`, and notifies both the PISP and the DFSP about these new changes
 to the Consent resource.
 
-![Credential registration: Register](http://www.plantuml.com/plantuml/proxy?src=https://raw.githubusercontent.com/mojaloop/pisp/master/docs/linking/5b-credential-registration-2.puml&cache=no)
+
+<!-- Note: this diagram is too big for the proxy renderer - so it is maintained locally -->
+![Credential registration: Register](../out/linking/5b-credential-registration/PISP%20Linking%20%20Credential%20registration%20(verification).png)
```
Comment (jgeewax) [inline docs/linking/README.md:233]: TODO: Can we make a post-commit hook or something that would auto-gen the diagrams?

**Review by jgeewax — APPROVED** [review]
This looks like what we discussed in Slack :-) LGTM.

File: docs/linking/README.md (hunk 1 above)
Comment (lewisdaly) [reply docs/linking/README.md:233]: ooh good idea

Comment (lewisdaly) [reply docs/linking/README.md:233]: created: mojaloop/project#1763

**Pull request merged**